en-GB.lproj/Main.strings

```diff
@@ -84,19 +84,19 @@
 "ELb-9J-7YK.normalTitle" = "placeholder";
 
 /* Class = "UILabel"; text = "Set CS_DEBUGGED"; ObjectID = "Eq9-Ns-zey"; */
-"Eq9-Ns-zey.text" = "Set CS_DEBUGGED";
+"Eq9-Ns-zey.text" = "Vestigen CS_DEBUGGED";
```

Comment:
"Stel CS_DEBUGGED in" is a more fitting term. Did "Vestigen" come from Google Translate?